tests/testthat/test-mock-api.R

```diff
@@ -212,6 +212,17 @@ public({
       )
     })
   })
+
+  # check that the functions are untraced
+  expect_false(inherits(httr:::body_config, "functionWithTrace"))
+  expect_false(inherits(curl::form_file, "functionWithTrace"))
+  expect_false(inherits(httr:::request_perform, "functionWithTrace"))
+  for (verb in c("GET", "PUT", "POST", "PATCH", "DELETE", "VERB", "RETRY")) {
+    fun <- getFromNamespace(verb, 'httr')
```

Comment:
So that we don't have to use `:::` to get them:

```suggestion
  for (fun_name in c("GET", "PUT", "POST", "PATCH", "DELETE", "VERB", "RETRY", "body_config", "request_perform")) {
    fun <- getFromNamespace(fun_name, "httr")
```